internal/cmd/kubernetes_test.go

```diff
@@ -57,6 +59,12 @@ func TestWriteKubeconfig(t *testing.T) {
 		},
 	}
 
+	configFileStats, err := os.Stat(kubeConfigPath)
+	assert.NilError(t, err)
+
+	permissions := configFileStats.Mode().Perm()
+	assert.Assert(t, permissions == 0o600) // kube config should not be world or group readable, only user read/write
```

Comment:
I have to check that his permissions test work on Windows also, its using Unix file permissions.

Reply:
`assert.Equal` would give better failure messages here